PhysicsTools/PatAlgos/python/slimming/slimmedMuons_cfi.py

```diff
@@ -11,7 +11,9 @@
     slimCaloVars = cms.string("1"),
     slimKinkVars = cms.string("1"),
     slimCaloMETCorr = cms.string("1"),
-    slimMatches = cms.string("0"),
+    slimMatches = cms.string("1"),
```

Comment:
this changes existing parts of the slimmedMuons and falls outside the scope of 92X, which I think is now limited only to plain additions.